web/themes/custom/recipe_theme/templates/page/page.html.twig

```diff
@@ -0,0 +1,67 @@
+{#
+/**
+ * @file
+ * Template for the main page.
+ */
+#}
+{% set page_classes = [
+  'page',
+  is_userpage ? 'page--user',
```

Comment:
Although the solution you provided in `page.inc` can be useful in many cases, core's `classy` theme provides a more general and reusable solution that is not specific for user pages. Please see `web/core/themes/classy/templates/layout/html.html.twig` line `29`.